### ISSUE TITLE
Possible Brazilian AY.43 novel sublineages with ORF1a:G1155C and S:N950D mutations

### ISSUE THREAD

**Issue by Gabitech35** [cov-lineages/pango-designation#319]
By Gabriela Ribeiro, Alex Ranieri Lima, Svetoslav Slavov (Butantan Network for Pandemic Alert of SARS-CoV-2 Variants, São Paulo)

We observed an interesting clustering of the AY.43 lineage which might be suggestive for emerging sub-lineages. The subclades of the main clade were characterized by the presence of two principal mutations: ORF1a: G1155C and S: N950D. Additionally, in the subcluster characterized by the mutation G:1155C (ORF1a) we observed another subgroup that was characterized by the ORF8: Q18. Further observations must be performed via genomic surveillance to see if an increase will be noted in the number of sequenced strains. What was also important is that all of the sequences were sequenced in São Paulo, Brazil, and were originating from the metropolitan zone of this city. 

Here are the phylogenetic trees with clusters with the defining mutations. 
![1](https://user-images.githubusercontent.com/85954118/141531613-01611edc-7c2d-4847-8a66-281956c93892.png)
![4](https://user-images.githubusercontent.com/85954118/141531644-ccb7f8f1-1f3d-4b1e-aebd-27103519099c.png)
![5](https://user-images.githubusercontent.com/85954118/141531647-78df680b-657a-45e9-9310-c10784cc8276.png)
![6](https://user-images.githubusercontent.com/85954118/141531648-54368f21-8f4b-482e-8999-84fe11e06b52.png)
![7](https://user-images.githubusercontent.com/85954118/141531649-6a8ddd70-4d21-460e-99c2-e22bde786e05.png)
![2](https://user-images.githubusercontent.com/85954118/141531652-a8ae0e0d-2199-47ad-87eb-bc8a740e45e2.png)
![3](https://user-images.githubusercontent.com/85954118/141531653-03da98e3-a83e-4cc6-af05-2ae9d8768a6e.png)

We also present here the observed mutation frequencies.

AY.43  ORF1a:G1155C
https://cov-spectrum.ethz.ch/explore/World/AllSamples/Past3M/variants?aaMutations=ORF1a%3AG1155C&pangoLineage=AY.43

AY.43 ORF8:Q18
https://cov-spectrum.ethz.ch/explore/World/AllSamples/Past3M/variants?aaMutations=ORF8%3AQ18&pangoLineage=AY.43

AY.43 S:N950D
https://cov-spectrum.ethz.ch/explore/World/AllSamples/Past3M/variants?aaMutations=S%3AN950D&pangoLineage=AY.43

**Comment by corneliusroemer** [comment]
The reversion at S:950 is in my experience most likely a sequencing error. This site is known to be problematic.

Which tree build are you sharing those screenshots from? I'd like to have a look. This site 1155C doesn't show up prominently on the most up to date Neherlab Brazil build. Maybe the sequences in question have been excluded due to QC problems?

![image](https://user-images.githubusercontent.com/25161793/141537488-bd1ee3d5-8d6b-4ccf-b437-0f6b13490c5e.png)

https://nextstrain.org/groups/neherlab/ncov/sa-brazil?branchLabel=aa&c=gt-ORF1a_1155

**Comment by corneliusroemer** [comment]
One issue is that Brazilian sequences seem to come in rather late, the most recent sequences we have are from a month ago. On covSpectrum it does look interesting. [covSpectrum](https://cov-spectrum.ethz.ch/explore/Brazil/AllSamples/Past6M/variants?aaMutations=ORF1a%3AG1155C%2CN%3A9L&pangoLineage=B.1.617.2*)

![image](https://user-images.githubusercontent.com/25161793/141538219-bfff4bc3-467c-4096-94a6-6baece9aeee7.png)

Here's an Usher build, generated from the integration with covSpectrum: 
![image](https://user-images.githubusercontent.com/25161793/141538475-b56e1e1e-fff8-4098-8c59-5fdec55e9fb7.png)






https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_4116b_ee0850.json?branchLabel=aa%20mutations&label=nuc%20mutations:G210T,C241T,C3037T,G4181T,C6402T,C7124T,C8986T,G9053T,C10029T,A11201G,A11332G,C14408T,G15451A,C15952A,C16466T,C18744T,C19220T,C21618G,T22917G,C22995A,A23403G,C23604G,C24130T,C25469T,T26767C,T27638C,C27752T,C27874T,A28299T,A28461G,G28881T,G28916T,G29402T,G29742T

**Comment by FedeGueli** [comment]
Interestingly Orf1a:G1155C is present in around 10% of AY.69 (dominant South Korean lineage).

**Comment by Gabitech35** [comment]
@corneliusroemer 

Thank you for the info

We are removing sensitive information from our data and as soon as possible we will share the phylogenetic tree.

The genetic sequences of this tree have not yet been deposited in Gisaid.

**Comment by alexranieri** [comment]
@corneliusroemer 
We removed some sensitive information from our sequence names and used the ncov pipeline again. The output json from this build is attached in the zip file. 
[ncov_EpiWeek_forty-one_forty-three_Butantan.zip](https://github.com/cov-lineages/pango-designation/files/7532281/ncov_EpiWeek_forty-one_forty-three_Butantan.zip)

We highlighted the interesting mutations in this txt file:
[new_AY43.txt](https://github.com/cov-lineages/pango-designation/files/7532282/new_AY43.txt)
 
@FedeGueli This is interesting! Since São Paulo is an important travelling hub in Brazil.

**Comment by chrisruis** [comment]
Thanks @Gabitech35 Looking through the AY.43 clade in the latest UShER tree, it looks like there are 2 quite large clades of Brazil sequences and quite a few small clades of Brazil sequences. The 2 large clades are large enough to warrant new lineages and we've added these in as AY.43.1 and AY.43.2 in [v1.2.96](https://github.com/cov-lineages/pango-designation/releases/tag/v1.2.96). We've started AY.43.1 on a branch with C12663T (Orf1ab:A4133V) and C25433T (Orf3a:T14I) and started AY.43.2 on a branch with G3728T (Orf1ab:G1155C), in both cases these branches seem to coincide with the introduction(s) into Brazil.

The smaller clades are too small to designate currently but if you notice any of them growing substantially let us know and we can add them in.
